setup.py

```diff
@@ -36,14 +36,14 @@
     "diffusers >= 0.31.0, < 0.32.0",
     "huggingface_hub >= 0.24.7",
     "sentence-transformers == 3.2.1",
+    "sentencepiece",
```

Comment:
Hi @regisss , is it ok to add the dependency here,  or is https://github.com/huggingface/optimum-habana/pull/1547/files a more preferred manner when a special dependency needed for a model?

Reply:
Yes I would rather do it the same way as in #1547.
In general, if a new dependency is required for many models, it will probably become a hard dependency in Trasnformers. Sentencepiece is an optional dependency in Transformers and only required for this model and Baichuan, so let's not add it to the lib dependencies.

Reply:
Got it, thank you! PR updated.